DisplaySettings/DisplaySettings.json

```diff
@@ -1308,7 +1370,7 @@
             }
         },
         "getVolumeLeveller": {
-            "summary": "Returns the current Volume Leveller setting",
+            "summary": "(Version 1)Returns the current Volume Leveller setting.\n \n### Event \n\n No Events.",
```

Comment:
The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.